.travis.yml

```diff
@@ -6,14 +6,16 @@ node_js:
     - "4.1"
 
 before_install:
+    - rvm install 2.2.2
```

Comment:
It seems that Travis does not support multiple languages yet:

https://github.com/travis-ci/travis-ci/issues/4090